addon/components/ownership-selection-modal/groups/template.hbs

```diff
@@ -0,0 +1,11 @@
+<label class="margin-top-xx-sm">Group</label>
+{{ownership-selection light=true entity=entity ownership=ownership}}
```

Comment:
can you double check if `ownership` is really updated or if it's only `entity.ownedBy` that is. if it's not updated or useless, you can get rid of it